.docs/content/1.concepts/0.index.md

```diff
@@ -0,0 +1,32 @@
+[![License: AGPL v3](https://img.shields.io/badge/License-AGPL_v3-green.svg)](https://www.gnu.org/licenses/agpl-3.0)
```

Comment:
This item must go on the Readme like in this project for exemple

https://github.com/nuxt/nuxt